src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -367,6 +369,35 @@ bool isWMAbsolutePositionGood()
                        std::end(wmAbsPosGood),
                        [&](const sf::String& name) { return name == windowManagerName; });
 }
+
+// Initialize raw mouse input
+void initRawMouse(::Display* disp)
+{
+    auto opcode = 0;
+    auto event  = 0;
+    auto error  = 0;
```

Comment:
```suggestion
    int opcode = 0;
    int event  = 0;
    int error  = 0;
```

I don't see any benefit from using type inference here. I believe we've always been quite conservative with the use of auto and type deduction in SFML -- i.e. only use it when it provides a benefit and doesn't obfuscate the type.